static/assets/js/entity-reference/app.js

```diff
@@ -0,0 +1,59 @@
+(function(){
+    "use strict";
+
+    var app = angular.module('entityReferenceApp', []);
+
+    app.controller('EntityReferenceCtrl', ['$http', EntityReferenceCtrl]);
+
+    app.component('entityReference', {
+        templateUrl: '/assets/views/entities.html',
+        controller: 'EntityReferenceCtrl as ctrl',
+        bindings: {
+            title: '@',
+            getStartedUrl: '@',
+            swaggerUrl: '@',
+            whiteList: '@',
```

Comment:
The whitelist and blacklist is a temporary measure to allow for the use of the directive even though swashbuckler generates too many entities. The constituent swagger file had many more entities than was probably needed. I recall you mentioned the library was a bit overzealous when creating entities. Figured it was an easy feature to implement and allow time to fix the other library.

The Peer to Peer API will not need to use the black or white listing.